src/App.css

```diff
@@ -1,38 +1,54 @@
-.App {
-  text-align: center;
+/* Navbar CSS */
+nav.navbar {
+  padding: 18px 0;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 9999;
+  transition:  0.32s ease-in-out;
+  background-image: url("./assets/img/header-image-2.png");
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.logo-img {
+  max-width: 100px;
+  height: auto;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+span.navbar-text {
   display: flex;
-  flex-direction: column;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 }
 
-.App-link {
-  color: #61dafb;
+/* Add this to your stylesheet, e.g., App.css */
+nav.navbar .navbar-nav .nav-link {  
```

Comment:
using css parent selector on nested elements will make it hard to read and to maintain
for this project let's keep it up until the children only 
example: parent is nav.navbar and children is navbar-nav
nav.navbar .navbar-nav  {